iosHackthonApp/Modules/Features/MealList/Sources/MealList/MealListViewModel.swift

```diff
@@ -40,9 +40,11 @@ public final class MealListViewModel: ObservableObject {
                            completionHandler: { meals, error in
             guard
                 let meals = meals,
-                error == nil else { return }
+                error == nil else {
+                assertionFailure(String(describing: error?.localizedDescription))
```

Comment:
Same as above really. Just helpful for knowing whether there's no meals or a problem

Reply:
I think it's okay to force unwrap the error here to make this more readable since you've already checked that it's not nil:

`assertionFailure(error!.localizedDescription)` or `assertionFailure(error?.localizedDescription ?? "")`